<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pinCheck</class>
 <widget class="QDialog" name="pinCheck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enter PIN</string>
  </property>
  <widget class="QLineEdit" name="pinEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>193</width>
     <height>28</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter PIN to continue:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
